tests/php/Service/RegisteredMethodManagerTest.php

```diff
@@ -93,6 +134,17 @@ public function testDeleteFromMember()
         $this->assertNull($manager->getFromMember($member, new BasicMathMethod()));
     }
 
+    public function testDeleteFromMemberSendsNotification()
+    {
+        /** @var Member&MemberExtension $member */
+        $member = $this->objFromFixture(Member::class, 'bob_jones');
+
+        $manager = RegisteredMethodManager::singleton();
+        $manager->deleteFromMember($member, new BasicMathMethod());
+
+        $this->assertEmailSent($member->Email);
```

Comment:
It might be worth asserting some of the subject line to ensure the correct email was sent - you can pass regex into the arguments e.g. `$this->assertEmailSend($member->Email, null, '/method was deleted/')`

Reply:
Are tests always run in English?

Reply:
We can assume they are, yes. SapphireTest sets the locale to the configured default locale which is en_US by default. If you want to be super safe you can add the same translation calls in your assertions as in the source class, but Guy's actually proposing to remove non-English translations from unit tests by default in an upcoming release. We don't usually bother with this in assertions, just check English

Reply:
OK neat :)

Reply:
Although this isn't probably the best place to continue this chat...
The default translation for a site isn't necessarily English - the second parameter to `_t()` as I understand it is the fallback - it's still possible for a translation file to override this. I think ideally one would have a test state for translation or set to a non-existent lang such as "SS_TEST" in order to trigger the known state (fallback text).

Reply:
Correct. You have two options:

* Include identical `_t()` calls in your assertions (we've done this before in unit tests)
* Assume it'll be English (we've done this before in unit tests)

Reply:
I've assumed English.